theme/_layouts/default.html

```diff
@@ -1,325 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Why do we want to get rid of this file altogether?

Reply:
Answered below.